src/App.js

```diff
@@ -1,10 +1,14 @@
-import Calculator from './Component/calculator';
+/* import { Link, Routes, Route } from "react-router-dom"; */
+/* import MyCalculator from './components/calculator/calculator';
+import MyQuotes from './components/quotes/quotes';
+import Home from "./components/home/home"; */
```

Comment:
- [ ] since they ain't implemented from the app.js file, kindly remove the commented code as a measure for clean code practice. Nice work
- [optional] you can remove the reportwebvital file since it's of no use to you. This will also make your work clean